src/caja-location-bar.c

```diff
@@ -592,30 +595,61 @@ caja_location_bar_set_location (CajaLocationBar *bar,
     caja_location_bar_update_label (bar);
 }
 
+static void
+override_background_color (GtkWidget *widget,
+                           GdkRGBA   *rgba)
+{
+    gchar          *css;
+    GtkCssProvider *provider;
+
+    provider = gtk_css_provider_new ();
+
+    css = g_strdup_printf ("entry { background-color: %s; background-image: none;}",
```

Comment:
Why background-image: none ?
The non-active entry should simply use 
entry:disabled {} from themes.